### PR TITLE
add Labyrinth75 

### PR DESCRIPTION
**Pull request by Liviturtle**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/labyrinth75/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    ),
+    [3] = LAYOUT(
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,\
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,  _______, \
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,  _______,\
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,\
+        _______, _______, _______,_______,_______,_______,_______,_______,_______, _______,  _______,  _______,  _______, _______,\
+        _______, _______, _______,_______,_______,_______,_______,_______,_______ \
+    
+    ),
```
Comment (ridingqwerty) [inline keyboards/labyrinth75/keymaps/default/keymap.c:54]:
```suggestion
```

These empty, inaccessible layers can be omitted from the default keymap.

You could also omit the trailing backslash from the first layer to pretty it up, if you wish.

File: keyboards/labyrinth75/labyrinth75.h [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    k51,   k42, k52, k43, k53, k44, k54, k45, k55, k46, k56, k47, k57, k48,  k58, \
+    
+    k62,    k72, k63, k73, k64, k74, k65, k75, k66, k76, k67, k77,    k68,   k78, \
+
+    k01,     k11, k21, k41, k61, k71, k81, k82, k83, k84, k85,   k86,   k87, k88, \
+    
+    k50,  k30,  k00,               k10,              k20,  k40,    k60, k70, k80  \
+
+*/
```
Comment (ridingqwerty) [inline keyboards/labyrinth75/labyrinth75.h:64]:
```suggestion
```

File: keyboards/labyrinth75/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [0] = LAYOUT(
+        KC_ESC,   KC_F1, KC_F2, KC_F3, KC_F4,   KC_F5, KC_F6, KC_F7, KC_F8,   KC_F9, KC_F10, KC_F11, KC_F12,   KC_PSCR, \
```
Comment (ridingqwerty) [inline keyboards/labyrinth75/keymaps/via/keymap.c:21]: The terminating backslashes in `keymap.c` are superfluous, but I don't believe they are a blocker to merge.

**Review by ridingqwerty — CHANGES_REQUESTED** [review]
Some minor fix-ups suggested.

File: keyboards/labyrinth75/keymaps/via/keymap.c (hunk 3 above)
Comment (Liviturtle) [reply keyboards/labyrinth75/keymaps/via/keymap.c]: I've removed them anyway, thanks

File: keyboards/labyrinth75/config.h [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x4f53
+#define PRODUCT_ID   0x464b
```
Comment (drashna) [inline keyboards/labyrinth75/config.h:24]:
```suggestion
#define VENDOR_ID    0x4F53
#define PRODUCT_ID   0x464B
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by ridingqwerty — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**